website/source/docs/platform/k8s/injector/index.html.md

```diff
@@ -0,0 +1,264 @@
+  Defaults to `false`.
+
+* `vault.hashicorp.com/agent-inject-status` - blocks further mutations
+  by adding the value `injected` to the pod after a successful mutation.
+
+* `vault.hashicorp.com/agent-inject-secret` - configures Vault Agent
+  to retrieve the secrets from Vault required by the container. The name of the
+  secret is any unique string after `vault.hashicorp.com/agent-inject-secret-`,
+  such as `vault.hashicorp.com/agent-inject-secret-foobar`. The value is the path
+  in Vault where the secret is located.
+
+* `vault.hashicorp.com/agent-inject-template` - configures Vault Agent
+  what template to use for rendering the secrets.  The name of the template is any
+  unique string after `vault.hashicorp.com/agent-inject-template-`, such as 
+  `vault.hashicorp.com/agent-inject-template-foobar`. This should map to the same
+  unique value provided in `vault.hashicorp.com/agent-inject-secret-`. If not provided,
+  a default generic template is used.
+
+* `vault.hashicorp.com/role` - configures the Vault role used by the Vault Agent
+  auto-auth method.  Required when `vault.hashicorp.com/agent-configmap` is not set.
+
+* `vault.hashicorp.com/agent-configmap` - name of the configuration map where Vault
+  Agent configuration file and templates can be found.
+
+* `vault.hashicorp.com/agent-pre-populate` - configures whether an init container
+  is included to pre-populate the shared memory volume with secrets prior to the
+  containers starting.
+
+* `vault.hashicorp.com/agent-pre-populate-only` - configures whether an init container
+  is the only injected container. If true, no sidecar container will be injected
+  at runtime of the pod.
+
+* `vault.hashicorp.com/agent-image` - name of the Vault docker image to use. This
+  value overrides the default image configured in the controller and is usually
+  not needed.  Defaults to `vault:1.3.1`.
+
+* `vault.hashicorp.com/service` - name of the Vault service to use.  This value
+  overrides the default service configured in the controller and is usually not
+  needed.
+
+* `vault.hashicorp.com/agent-limits-cpu` - configures the CPU limits on the Vault
+  Agent containers.  Defaults to `500m`.
+
+* `vault.hashicorp.com/agent-limits-mem` - configures the memory limits on the Vault
+  Agent containers.  Defaults to `128Mi`.
+
+* `vault.hashicorp.com/agent-requests-cpu` - configures the CPU requests on the
+  Vault Agent containers.  Defaults to `250m`.
+
+* `vault.hashicorp.com/agent-requests-mem` - configures the memory requests on the
+  Vault Agent containers.  Defaults to `64Mi`.
+
+* `vault.hashicorp.com/tls-secret` - name of the Kubernetes secret containing TLS
+  Client and CA certificates and keys.  This is mounted to `/vault/tls`.
+
+* `vault.hashicorp.com/tls-server-name` - name of the Vault server to verify the
+  authenticity of the server when communicating with Vault over TLS.
+
+* `vault.hashicorp.com/tls-skip-verify` - configures the Vault Agent to verify
```

Comment:
"verify", or "skip verification of"?  (And if the latter, probably our usually disclaimer about this type of setting.)

Reply:
Fixed!